src/NuGet.Core/NuGet.Protocol/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,30 @@
 #nullable enable
+NuGet.Protocol.ClientVersions
+NuGet.Protocol.RegistrationsBaseUrlTypes
+~static readonly NuGet.Protocol.ClientVersions.Version200 -> string
+~static readonly NuGet.Protocol.ClientVersions.Version300 -> string
+~static readonly NuGet.Protocol.ClientVersions.Version300beta -> string
+~static readonly NuGet.Protocol.ClientVersions.Version300rc -> string
+~static readonly NuGet.Protocol.ClientVersions.Version340 -> string
+~static readonly NuGet.Protocol.ClientVersions.Version360 -> string
+~static readonly NuGet.Protocol.ClientVersions.Version470 -> string
+~static readonly NuGet.Protocol.ClientVersions.Version490 -> string
+~static readonly NuGet.Protocol.ClientVersions.Version500 -> string
+~static readonly NuGet.Protocol.ClientVersions.Version510 -> string
+~static readonly NuGet.Protocol.ClientVersions.Versioned -> string
+*REMOVED*~static readonly NuGet.Protocol.ServiceTypes.Version200 -> string
+*REMOVED*~static readonly NuGet.Protocol.ServiceTypes.Version300 -> string
+*REMOVED*~static readonly NuGet.Protocol.ServiceTypes.Version300beta -> string
+*REMOVED*~static readonly NuGet.Protocol.ServiceTypes.Version340 -> string
+*REMOVED*~static readonly NuGet.Protocol.ServiceTypes.Version470 -> string
+*REMOVED*~static readonly NuGet.Protocol.ServiceTypes.Version490 -> string
+*REMOVED*~static readonly NuGet.Protocol.ServiceTypes.Version500 -> string
+*REMOVED*~static readonly NuGet.Protocol.ServiceTypes.Version510 -> string
+*REMOVED*~static readonly NuGet.Protocol.ServiceTypes.Versioned -> string
```

Comment:
Git already tracks that APIs were removed in PublicAPI.Shipped.txt, so I don't think this really adds anything.  Additionally, does the PublicApiAnalyzer even work with these lines in this file?